### DISCUSSION TITLE
Adding NX functionality via my fork.

### DISCUSSION THREAD

**Discussion by FunnyCorgi** [copy/v86#1017] [General]
I will be posting regular updates here, and will be asking questions about the existing code as well.
https://github.com/FunnyCorgi/v86

**Comment by FunnyCorgi** [comment]
how does V86 interact with memory? I need to know this so I can make a custom nx section in the cpu that will store which bits in RAM are executable(which is what NX does).

**Reply by spetterman66** [reply]
writeups on nx-bit:
https://h10032.www1.hp.com/ctg/Manual/c00317515.pdf
https://web.archive.org/web/20170109184919/http://h21007.www2.hpe.com/portal/StaticDownload?attachment_ciid=d861e0aecd2b7110VgnVCM100000275d6e10RCRD&ciid=1cd124be1d0c7110VgnVCM100000275d6e10RCRD

v86 memory interaction:
https://github.com/copy/v86/blob/master/src/rust/cpu/memory.rs
